frontend/src/metabase-lib/v1/Question.ts

```diff
@@ -324,22 +324,6 @@ class Question {
     return this.setDisplay(display).updateSettings(settings);
   }
 
-  /**
-   * This function is used to conditionally avoid calling Lib.ensureFilterStage for pivoted questions.
-   * Pivot tables cannot work when there is an extra stage added on top of breakouts and aggregations.
-   */
-  isPivoted(): boolean {
```

Comment:
Now we will only be checking for `display === "pivot"`.
Before we also checked for pivoted `display === "table"` because it helped preserve viz settings in some cases (when drilling).